conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureHandler.java

```diff
@@ -83,11 +98,19 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
     }
 
     private void register(Endpoint endpoint) {
-        HttpHandler current = endpoint.handler();
-        for (BiFunction<Endpoint, HttpHandler, HttpHandler> wrapper : WRAPPERS) {
-            current = wrapper.apply(endpoint, current);
+        Endpoint current = endpoint;
+        current = stackEndpointHandlerWrapper(current, WRAPPERS_AFTER_BLOCKING);
+        current = stackEndpointHandlerWrapper(current, wrappersJustBeforeBlocking.reverse());
+        current = stackEndpointHandlerWrapper(current, WRAPPERS_BEFORE_BLOCKING);
```

Comment:
What if we construct a single EndpointHandlerWrapper in `Builder.build()` that we use once here?

```java
private void register(Endpoint endpoint) {
    routingHandler.add(endpoint.method(), endpoint.template(), wrapper.wrap(current));
}
```